### DISCUSSION TITLE
Will you be able to add this for windows 11 Virtual Desktops?

### DISCUSSION THREAD

**Discussion by pouyarezvani** [noah-nuebling/mac-mouse-fix#172] [Questions] [answered]
I have been using your v2 Beta on my Mac for the past few months, and I can't live without it now. 
However, I recently upgraded to windows 11 and virtual desktops on Windows 11 are hugely updated to look like mac. 
I was wondering if it is possible to create this same product for windows 11 users? 
not sure if you can, but if you do, you are literally boosting our productivity. 
Thanks! 

Love from San Francisco, California.

**Comment by noah-nuebling** [comment, marked as answer]
That would be super cool, but you'd have to redo like 70% of the app from scratch. And the remaining 30% of internal logic still uses Apple libraries for arrays, dictionaries, etc. So probably not coming any time soon or ever, sorry :|

Oh and thanks so much for the kind words, glad you're liking the app! :)

**Reply by pouyarezvani** [reply]
Our ENTIRE team at the office uses your software. We all use macs at work, and your app is an incredible productivity booster for us. 
I don't like using a trackpad and I'm a more mouse fan, so your app just nails everything the trackpad offers. There are some small little bugs I've come across sometimes, but otherwise very good. 
Thanks.

**Reply by noah-nuebling** [reply]
Hey @pouyarezvani, thanks!! I'm so happy to hear that. That's really cool. 
Btw [version 2.1](https://github.com/noah-nuebling/mac-mouse-fix/releases) is out now!

**Reply by noah-nuebling** [reply]
Marking this as answered to clean things up. Let me know if you have more questions!

**Comment by EricxWood** [comment]
Hi @pouyarezvani, I'm working on a project that provide smooth scroll on windows and will be release in some days. There's a demo program in my repo Scroil right now, just a algorithm demo, I'll be working on other features recently.
